test/gtest/ucs/test_memtrack.cc

```diff
@@ -175,9 +175,21 @@ UCS_TEST_F(test_memtrack, memalign_realloc) {
     ptr = ucs_realloc(ptr, 2*ALLOC_SIZE, ALLOC_NAME);
     ASSERT_NE((void *)NULL, ptr);
 
+    ret = ucs_posix_memalign_realloc(&ptr, 1024, ALLOC_SIZE, ALLOC_NAME);
+    ASSERT_EQ(0, ret);
+    ASSERT_NE((void *)NULL, ptr);
+
+    ret = ucs_posix_memalign_realloc(&ptr, 1024, 4*ALLOC_SIZE, ALLOC_NAME);
+    ASSERT_EQ(0, ret);
+    ASSERT_NE((void *)NULL, ptr);
+
+    ret = ucs_posix_memalign_realloc(&ptr, 1<<20, 2*ALLOC_SIZE, ALLOC_NAME);
+    ASSERT_EQ(0, ret);
+    ASSERT_NE((void *)NULL, ptr);
+
     ucs_free(ptr);
 
-    test_total(1, 2 * ALLOC_SIZE);
+    test_total(1, 4*ALLOC_SIZE);
```

Comment:
1. use space around `*` and `<<` math operators, and no space between void and *: `(void*)`
2. check the returned pointer is really aligned to the desired boundary
3. use UCS_KBYTE/UCS_MBYTE constatnts